test/plsql.test.ts

```diff
@@ -164,7 +165,7 @@ describe('PlSqlFormatter', () => {
     `);
     expect(result).toBe(dedent`
       WITH
-        t1(id, parent_id) AS (
+        t1 (id, parent_id) AS (
```

Comment:
Not really sure which way is better here. It's not a function call.

However it should be possible to improve this with special parsing of WITH clause in the future.